tabpy/tabpy_server/app/app.py

```diff
@@ -171,28 +173,37 @@ def set_parameter(settings_key,
                           config_key,
                           default_val=None,
                           check_env_var=False):
+            key_set = False
```

Comment:
why use a flag instead of else ifs? this variable is confusing, I expected it to be a set not a bool

Reply:
nested if's - this is why. renaming to key_is_set